qkernel/src/kernel_def.rs

```diff
@@ -396,6 +396,11 @@ impl HostSpace {
 
 #[inline]
 pub fn child_clone(userSp: u64) {
+    #[cfg(target_arch = "aarch64")] {
```

Comment:
I'm not sure if we need to flush TCB here. During the during the context_swap_to the TCB should already be flushed.